wordpress-mobile/WordPress-iOS
# Add analytics for plugin usage

I've tried to closely match @oguzkocer Android implementation in https://github.com/wordpress-mobile/WordPress-Android/pull/6994

See #8318 

To test:

You can set a breakpoint on WPAnalyticsTrackerAutomatticTracks.m:71 and log `📡 @eventPair.eventName@: @mergedProperties.description@`

![screen shot 2017-12-15 at 09 45 40](https://user-images.githubusercontent.com/8739/34033974-cf7dbd80-e17c-11e7-9fd8-6165d3f6f7a5.png)

Then check that it tracks:
- When you open the plugins section
- When you update a plugin
- When you remove a plugin
